bench/bench.go

```diff
@@ -36,7 +37,16 @@ func getClusterVersion(ctx context.Context, hosts []string) (string, error) {
 	if len(hosts) == 0 {
 		return "", fmt.Errorf("can't get cluster version with no pilosa hosts configured")
 	}
-	resp, err := http.Get("http://" + hosts[0] + "/version")
+	clientOptions := ctx.Value("clientOptions").(*pcli.ClientOptions)
```

Comment:
I'd prefer to add an argument rather than use context.Value - this just adds hidden API which becomes a nightmare if we ever try to refactor the code.

Reply:
Added `HostSetup` struct.